### PR TITLE
use departure timeout

### PR DESCRIPTION
**Pull request by paulwe**

### CODE REVIEW

File: pkg/rtc/room.go [hunk 1]
```diff
@@ -774,7 +774,7 @@ func (r *Room) CloseIfEmpty() {
 	if r.FirstJoinedAt() > 0 && r.LastLeftAt() > 0 {
 		elapsed = time.Now().Unix() - r.LastLeftAt()
 		// need to give time in case participant is reconnecting
-		timeout = RoomDepartureGrace
```
Comment (boks1971) [inline pkg/rtc/room.go:777]: Can we delete this constant? Guess it is used in some test. Wondering if the test can be changed to not use that constant. Would be good to get rid of constants that might be confusing.

**Review by boks1971 — APPROVED** [review]

File: pkg/config/config.go [hunk 2]
```diff
@@ -232,6 +232,7 @@ type RoomConfig struct {
 	EnabledCodecs      []CodecSpec        `yaml:"enabled_codecs,omitempty"`
 	MaxParticipants    uint32             `yaml:"max_participants,omitempty"`
 	EmptyTimeout       uint32             `yaml:"empty_timeout,omitempty"`
+	DepartureTimeout   uint32             `yaml:"departure_timeout,omitempty"`
```
Comment (boks1971) [inline pkg/config/config.go:235]: Probably need to update `config-sample.yaml` with this.

File: pkg/rtc/room_test.go [hunk 3]
```diff
@@ -737,6 +735,10 @@ func newRoomWithParticipants(t *testing.T, opts testRoomOpts) *Room {
 		&livekit.Room{Name: "room"},
 		nil,
 		WebRTCConfig{},
+		config.RoomConfig{
+			EmptyTimeout:     5 * 60,
+			DepartureTimeout: 1,
```
Comment (davidzhao) [inline pkg/rtc/room_test.go:740]: should we change the default to be higher? i.e. 20s?

Comment (boks1971) [reply pkg/rtc/room_test.go:740]: This is a test file. Config.go has the default of 20s

Comment (davidzhao) [reply pkg/rtc/room_test.go:740]: oops.. misread.. myb

**Pull request merged**